dts/arm/nxp/nxp_rt1015.dtsi

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2019, Linaro
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nxp/nxp_rt.dtsi>
+
+/* i.MX rt1015 default FlexRAM partition:
+ * ITCM: 32KB
+ * DTCM: 32KB
+ * OCRAM: 64KB
+ */
+&itcm0 {
+	compatible = "nxp,imx-itcm";
```

Comment:
You don't need the compatible here, its already in nxp_rt.dtsi.  Remove from all other nodes as well.

Reply:
thank you, will do.